astropy/units/tests/test_quantity_array_methods.py

```diff
@@ -122,6 +125,22 @@ def test_swapaxes(self):
         assert q_swapaxes.unit == q.unit
         assert np.all(q_swapaxes.value == q.value.swapaxes(0, 2))
 
+    def test_flat(self):
```

Comment:
My tendency would be to keep the tests together, i.e., move this to just below the other `test_flat` and rename it `test_flat_attributes`.

Reply:
Sounds good.